tests/common.rs

```diff
@@ -37,3 +37,28 @@ pub fn setup_pie() -> CairoPie {
 
     runner.get_cairo_pie(&vm).unwrap()
 }
+
+#[allow(unused)]
```

Comment:
why is it flag as `unused` ?

Reply:
no clue but it's very annoying. But i don't want to have it in the same file as the tests